pcap_set_fanout_linux.3pcap

```diff
@@ -0,0 +1,65 @@
+.\" Copyright (c) 1994, 1996, 1997
+.\"	The Regents of the University of California.  All rights reserved.
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that: (1) source code distributions
+.\" retain the above copyright notice and this paragraph in its entirety, (2)
+.\" distributions including binary code include the above copyright notice and
+.\" this paragraph in its entirety in the documentation or other materials
+.\" provided with the distribution, and (3) all advertising materials mentioning
+.\" features or use of this software display the following acknowledgement:
+.\" ``This product includes software developed by the University of California,
+.\" Lawrence Berkeley Laboratory and its contributors.'' Neither the name of
+.\" the University nor the names of its contributors may be used to endorse
+.\" or promote products derived from this software without specific prior
+.\" written permission.
+.\" THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR IMPLIED
+.\" WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED WARRANTIES OF
+.\" MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+.\"
+.TH PCAP_SET_FANOUT 3PCAP "16 April 2020"
+.SH NAME
+pcap_set_fanout_linux \- set fanout group for load balancing
+among processes
+.SH SYNOPSIS
+.nf
+.ft B
+#include <linux/if_packet.h>
+#include <pcap/pcap.h>
+.LP
+.ft B
+int pcap_set_fanout_linux(pcap_t *p, int enable, uint16_t mode, uint16_t group_id);
+.ft
+.fi
+.SH DESCRIPTION
+On Linux network interface devices,
+.B pcap_set_fanout_linux()
+sets the fanout configuration to be used in the
```

Comment:
Could you please explain what the affect of having packet fanout mode?
When would I turn it on and why?  Is there is a downside to it?  What if every application turned it on?

Reply:
In the AF_PACKET fanout mode, packet reception can be load balanced among processes/threads.

When "configuring" the socket to be used, you would be required to set a group_id, use the same group_id to achieve the required load balancing.

Reply:
Don't tell me. Put it in the manpage!